app.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+
+// Handle UI task
+
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('.table');
+    const row = document.createElement('tr');
+    const tableData = document.createElement('td');
+    tableData.classList.add('titleList');
+    row.appendChild(tableData);
+    tableData.innerHTML = title.value;
+    const tableDataTwo = document.createElement('td');
+    tableDataTwo.classList.add('authorList');
+    row.appendChild(tableDataTwo);
+    tableDataTwo.innerHTML = author.value;
+    const tableDataThree = document.createElement('td');
+    row.appendChild(tableDataThree);
+    const tdBtn = document.createElement('button');
+    tdBtn.classList.add('delete');
+    tdBtn.innerHTML = 'Remove';
+    tableDataThree.appendChild(tdBtn);
+    list.appendChild(row);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+}
+
+// Store class: Handle storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    // books.push(book);
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+// event display book
+document.addEventListener('DOMContentloaded', UI.displayBooks());
+// Event Add a book
+document.querySelector('#book-form').addEventListener('submit', (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+  // Instatiate book
+  const book = new Book(title, author);
+  // Add book to UI
+  UI.addBookToList(book);
+  // Add book to store
+  Store.addBook(book);
+  // clear fields
+  UI.clearFields();
+});
```

Comment:
- [ ] Your books are not really persisted in the local storage (see image). Kindly fix that.

  ![anna-local](https://user-images.githubusercontent.com/36542986/171366961-9b34d0af-aa6f-45ac-a3e4-53652740b4cd.jpg)

- [ ] **REMINDER:** After the books are persisted, kindly ensure that after the page reload those same books from the localstorage is loaded and rendered on the page.